ggshield/core/config/user_config.py

```diff
@@ -169,17 +169,23 @@ class IaCConfig(FilteredConfig):
     ignored_paths: List[IaCConfigIgnoredPath] = field(default_factory=list)
     ignored_policies: List[IaCConfigIgnoredPolicy] = field(default_factory=list)
     minimum_severity: str = "LOW"
+    outdated_ignored_paths: List[IaCConfigIgnoredPath] = field(default_factory=list)
+    outdated_ignored_policies: List[IaCConfigIgnoredPolicy] = field(
```

Comment:
I am not fond of using a `*Config` class to store computed data. As the docstring of the class say, this class is supposed to store the content of the IAC config of a .gitguardian.yaml file. But the border is fuzzy here: the Config class itself does too much already IMO.

If we keep these fields, they must be marked as excluded in the UserConfig schema, like we did for `deprecation_messages` (another ugly hack). Otherwise they will be serialized if the user runs one of the commands that write to the .gitguardian.yaml file (such as `ggshield secret ignore --last-found`)

According to [Marshmallow doc](https://marshmallow.readthedocs.io/en/stable/api_reference.html#marshmallow.Schema), you should be able to refer to these fields from the UserConfig schema using dot delimiters.

Reply:
Okay, let's keep it that way then. I did add a comment, isn't it what you meant?

Reply:
Alright, thanks for your review!